src/components/alert/docs.mdx

```diff
@@ -0,0 +1,100 @@
+---
+componentName: 'Alert'
+---
+
+The `Alert` component is used in long-form MDX content to highlight various message types. Default titles and icons are provided for each 'type', however these can be overridden via the `title` prop.
+
+## Props
+
+| Prop name  | Description                                                            | Required? | Default value      | Type                             |
+| ---------- | ---------------------------------------------------------------------- | --------- | ------------------ | -------------------------------- |
+| `children` | The alert body text to render.                                         | Yes       | undefined          | string                           |
+| `type`     | The 'type' which has associated styles and default data (icon, title). | No        | tip                | tip, highlight, warning, or note |
+| `title`    | An option to override the default title.                               | No        | The type's default | string                           |
+
+## Playground
+
+<LiveComponent>
+	{`
+	<Alert type="highlight" title="This is a custom title">
+		Caramels tart candy canes shortbread lemon drops.	
+	</Alert>
+	`}
+</LiveComponent>
+
+## Alert types
+
+### Tip
+
+<Alert>
```

Comment:
I asked @Jasmine-hashi to get education teams feedback on the authoring interface. Either they use `Alert` and set the type as a prop. Or we could export `Tip`, `Note`, `Highlight`, `Warning` to be used directly. Waiting to hear back from them. But any opinions are appreciated.